src/eterna/pose3D/Pose3DWindow.ts

```diff
@@ -391,10 +435,17 @@ export default class Pose3DWindow extends ContainerObject implements MouseWheelL
 
         // Title bar drag handles should fill remaining space
         const remainingWidth = this._currentBounds.width - (
-            this.ICON_SIZE * 2 + this.GAP * 6 + this._titleText.display.width + this._dropdown.width
+            this.ICON_SIZE * 2 + this.GAP * 5 + this._titleText.display.width + this._dropdown.width
         );
-        this._titleDraggerLeft.display.width = remainingWidth / 2;
-        this._titleDraggerRight.display.width = remainingWidth / 2;
+        const lW = this.GAP * 3 + this.ICON_SIZE + this._dropdown.width;
+        const tX = this._currentBounds.width / 2 - this._titleText.display.width / 2;
```

Comment:
What are `lW` and `tX`? Please use more descriptive names

Reply:
fixed